cmd/github-migration/v1/github-migration.go

```diff
@@ -38,28 +41,75 @@ func Execute() { // hello
 		// sync
 		appSync        = app.Command("sync", "Sync Bitbucket and GitHub repositories")
 		appSyncRunOnce = appSync.Flag("run-once", "Syncs the repositories once").Bool()
-		// appSyncPersonalAccount    = appSync.Flag("personal-account", "Migrates/Syncs the repositories to personal GitHub account").Bool()
+		appSyncPersonalAccount    = appSync.Flag("personal-account", "Migrates/Syncs the repositories to personal GitHub account").Bool()
 		appSyncBlockNewMigrations = appSync.Flag("block-new-migrations", "Block new migrations and sync only existing repos on GitHub").Bool()
 
 		/////////
 		// interactive - Can leave it out, does not make sense if supporting multiple sources for integrations
 		appInteractive = app.Command("interactive", "Select the projects and repositories to migrate/sync")
+
+		/////////
+		// test
+		appTest = app.Command("test", "Test out new features")
 	)
 
 	p := kingpin.MustParse(app.Parse(os.Args[1:]))
 
+	config := config.Parse()
+	fmt.Printf("%+v\n", config)
+
 	switch p {
 
 	case appSync.FullCommand():
-		fmt.Println("SYNC")
-		fmt.Printf("App Log Level: %v\n", *appLogLevel)
-		fmt.Printf("Run Once: %v\n", *appSyncRunOnce)
-		// fmt.Printf("Personal Account: %v\n", *appSyncPersonalAccount)
-		fmt.Printf("Block New: %v\n", *appSyncBlockNewMigrations)
+		log.Infof("SYNC")
+		log.Infof("App Log Level: %v\n", *appLogLevel)
+		log.Infof("Run Once: %v\n", *appSyncRunOnce)
+		fmt.Printf("Personal Account: %v\n", *appSyncPersonalAccount)
+		log.Infof("Block New: %v\n", *appSyncBlockNewMigrations)
 
 	case appInteractive.FullCommand():
-		fmt.Println("INTERACTIVE")
-		fmt.Printf("App Log Level: %v\n", *appLogLevel)
+		log.Infof("INTERACTIVE")
+		log.Infof("App Log Level: %v\n", *appLogLevel)
+	
+	case appTest.FullCommand():
+		log.Infof("TEST")
+		var input plugins.Input
+		var output plugins.Output
+		for _, integration := range config.Integrations {
+			fmt.Println(integration.Name)
+
+
+			// get input plugin based on input type
+			switch integration.Source.Type {
+			case "bitbucket-cloud":
+				input = bbcloud.New(integration.Source)
+			}
+			// Authenticate credentials for reading from input
+			_, err := input.Authenticate()
+			if err != nil {
+				log.Errorf(err.Error())
+			}
+			// Get repositories to sync
+			repos, err := input.Repositories(true)
+			if err != nil { log.Errorf(err.Error()) }
+
+
+			// get output plugin based on output type
+			switch integration.Target.Type {
+			case "github-public":
+				output = ghpublic.New(integration.Target)
+			}
```

Comment:
SHOULD add a "default" case that errors out, in case the user typo's a setting in the config file.